serverless/telegram-handler/src/credentials.ts

```diff
@@ -0,0 +1,60 @@
+import AWS from 'aws-sdk'
+import { Sequelize } from 'sequelize-typescript'
+
+import config from './config'
+import { Logger } from './utils/logger'
+
+const logger = new Logger('credentials')
+
+const secretsManager = new AWS.SecretsManager({
+  region: config.get('aws.awsRegion'),
+})
+
+/**
+ * Verifies that the given bot id is registered, otherwise throws an error.
+ */
+export const verifyBotIdRegistered = async (
+  botId: string,
+  sequelize: Sequelize
+): Promise<void> => {
+  const { exists: botIdExists } = await sequelize.query(
+    `SELECT EXISTS (SELECT * FROM credentials WHERE name = :botId);`,
```

Comment:
Strange syntax :/

`SELECT 1 AS exists FROM credentials WHERE name = :botId;`
will return you exists = undefined if it doesn't exist, and exists = 1 if it does.